### PR TITLE
Update steps to create repo secrets

### LINKED ISSUE #17791: "Creating encrypted secrets for a repository" in "Encrypted Secrets" page is outdated
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/actions/security-guides/encrypted-secrets#creating-encrypted-secrets-for-a-repository

### What part(s) of the article would you like to see updated?

Right now are the mentioned steps as follows:

1. On GitHub.com, navigate to the main page of the repository.
2. Under your repository name, click **Settings**.
   image
3. In the left sidebar, click **Secrets**.
4. Click **New repository secret**.
5. Type a name for your secret in the **Name** input box.
6. Enter the value for your secret.
7. Click **Add secret**.

The list is outdated by having the **Secrets** field now open a selection of options (Actions, Codespaces and Dependabot) rather than directly leading to a page where you can create the Repository Secret.

image

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by thispsj**
### Why:

Closes #17791

### What's being changed:

 
Everything mentioned by @github-actions in Content Changes table. In Dependabot article, removed one step which was not needed.

233604b - Updated the reusable to the latest steps 

### Check off the following:

- [x] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [X] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Comment by janiceilene** [comment]
@thispsj Thanks so much for opening a PR! I'll get this triaged for review ✨

**Comment by mchammer01** [comment]
I'll review this as I need to check the behavior on GHES 3.3, 3.4 and 3.5.

**Review by mchammer01 — COMMENTED** [review]
@thispsj 👋🏻 - thanks for your contribution ✨ 

I've made some suggestions as I tested  the behavior on dotcom, GHES 3.3, GHES 3.4 and GHES 3.5.
 I'd like to edit the `{% data reusables.actions.sidebar-secret %}` reusable (found in data/reusables/dependabot/dependabot-secrets-button) but I wasn't able to fetch your branch and add a commit.
I'd like to edit it so it contains the following:

```
 {% ifversion fpt or ghec or ghes > 3.2 or ghae-issue-5658 %}
1. Inside the "Secrets" section of the sidebar, click **{% data variables.product.prodname_dependabot %}**.
   ![{% data variables.product.prodname_dependabot %} secrets sidebar option](/assets/images/enterprise/3.3/dependabot/dependabot-secrets.png)
{% endif %}
```

Can you help with that? (Not sure why I can't get your branch). Thanks!

**Comment by Philepe25** [comment]
Ok

File: data/reusables/actions/sidebar-secret.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {% ifversion fpt or ghec or ghes > 3.4 or ghae-issue-5658 %}
-1. In the left sidebar, click **{% octicon "key-asterisk" aria-label="The key-asterisk icon" %} Secrets**.
+1. In the "Security" section of the sidebar, select **{% octicon "key-asterisk" aria-label="The key-asterisk icon" %} Secrets**, then click **Actions**.
```
[the remark this answers is not in the document]
Comment (thispsj) [reply data/reusables/actions/sidebar-secret.md:2]: Hii @mchammer01,

I don't why you aren't able to access my branch. I will incorporate all those changes.

Before doing that a small question, I will revert back the reusable, after that I should add one more step in `actions/security-guides/encrypted-secrets.md` to select Actions from the expanded secrets menu, right ?

[the remark this answers is not in the document]
Comment (thispsj) [reply data/reusables/actions/sidebar-secret.md:2]: > Hi @thispsj 👋🏻 - I don't think we need to make that change as my understanding is that once you click **Secrets**, you'll have to click **Dependabot**, not **Actions,** as the procedural section explains how to add a repository secret **for Dependabot**.
> 
> dependabot-secrets
> 
> Let me know if what I'm saying doesn't make sense 🙂

Hello again @mchammer01 ,

Ya you're right, but I'm talking about `actions/security-guides/encrypted-secrets.md` this section talks about adding secrets for GitHub Actions.

**Comment by mchammer01** [comment]
{% data reusables.actions.sidebar-secret %} is used in 4 files:
1. content/actions/security-guides/encrypted-secrets.md (3 occurrences) 
2. content/actions/using-workflows/sharing-workflows-secrets-and-runners-with-your-organization.md (1 occurrence)
3. content/code-security/dependabot/working-with-dependabot\managing-encrypted-secrets-for-dependabot.md (2 occurrences)
4. /content/codespaces/managing-codespaces-for-your-organization/managing-encrypted-secrets-for-your-repository-and-organization-for-codespaces.md (1 occurence)

1 and 2 relate to **Actions**, 3 to **Dependabot** and 4 to **Codespaces**.

@thispsj  👋🏻 - so my recommendation would be to:

A- Update {% data reusables.actions.sidebar-secret %} so that it mentions clicking "Actions" inside "Secrets" (like you initially did in this PR). This takes care of 1 and 2. We can no longer use this reusable for 3 and 4 as it now specifically mentions Actions.

B. Add 2 new reusables, one in `data/reusables/dependabot` called sidebar-secret (so you'd use {% data reusables.dependabot.sidebar-secret %} when calling the reusable) and the other in `data/reusables/codespaces` (so you'd use {% data reusables.codespaces.sidebar-secret %} when calling the reusable), which should be used in the files involved in 3 and 4. In one, we instruct the user to select "Secrets" on the left sidebar, then click "Dependabot" inside the "Secrets" section, whereas for the the other one, we tell them to select "Secrets" on the left sidebar, then click "Codespaces"  inside the "Secrets" section.

What do you think? Do you feel comfortable with implementing this?

**Comment by mchammer01** [comment]
@thispsj 👋🏻 - I'm going to be away for a few days so once your new updates are ready, would you mind re-adding the `waiting for review` label to this PR, so that it gets added to our review board? Thank you 🙂

**Comment by thispsj** [comment]
Hi @mchammer01 ,
How can I re label `waiting for review` ? I don't have triage access to this repo.

**Comment by mchammer01** [comment]
@janiceilene / @cmwilson21 - could you put this up for review once @thispsj has made the suggested changes? Thank youuu 🙇🏻 😄

**Comment by thispsj** [comment]
@mchammer01 good idea, I'll just tag one of them once I'm done.

**Comment by thispsj** [comment]
@cmwilson21 :wave:
This PR is ready for a review now !
Please add the `waiting for review` label so that it gets added to the review board.

**Review by Vana12330 — COMMENTED** [review]
0x8463a980e5b1fe116df96f8672dae9934b131cb5

**Review by lucascosti — APPROVED** [review]
Sorry for the delay on this one! I've resolved the conflicts and all looks good. I'll get this merged! 🚀

**Comment by thispsj** [comment]
> Sorry for the delay on this one! I've resolved the conflicts and all looks good. I'll get this merged! 🚀

@lucascosti no problem 🙂

**Comment by lohva** [comment]
hhhhhmmmmm     ^_^     and     $_$

**Pull request merged**